grammar.js

```diff
@@ -226,7 +224,7 @@ module.exports = grammar({
       $.argument_pair
     ))),
 
-    argument_pair: $ => prec(PREC.KW_ARG, seq(choice(
+    argument_pair: $ => prec(-1, seq(choice(
```

Comment:
Temporary until we can re-use the `pair` production without blowing up generate times and causing an explosion in parse states.